### DISCUSSION TITLE
[General] Burn address

### DISCUSSION THREAD

**Discussion by ghost** [zkSync-Community-Hub/zksync-developers#36] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

Hello,

I would like to burn some useless coins on zksync lite. What is the burn address?

Thanks

### What is the specific issue or error you're encountering?

n/a

### Can you share the error messages or logs you're receiving, if any?

n/a

### Have you made any recent changes to the contract before encountering this issue?

n/a

### Are there any external libraries or contracts that your contract interacts with?

n/a

### Can you provide the relevant portions of your contract code where the issue is occurring?

n/a

### Have you tried to isolate the problem, and if so, what were the results?

n/a

### What steps have you already taken to try to resolve the issue?

Tried googling

**Comment by dutterbutter** [comment, marked as answer]
Hey 👋 You should be able to make use of Ethereums common burn address `0x0000000000000000000000000000000000000000`.

**Comment by steyn52** [comment]
The burn address for zkSync is typically an Ethereum address where tokens are sent to be permanently removed from circulation. However, zkSync does not have a publicly known burn address like some other blockchains. Instead, tokens may be sent to an inaccessible address or a zero-address (0x000...). It's important to confirm with zkSync's documentation or community for the most accurate information regarding burning tokens.

**Comment by evmeiwear** [comment]
The burn address for zkSync is 0x0000000000000000000000000000000000000000. You can send tokens to this address if you want to burn them.

**Comment by mcleodjv** [comment]
The burn address for zkSync on the Mainnet is 0x0000000000000000000000000000000000000000. You can send tokens to this address if you want to permanently remove them from circulation.

**Comment by casesumteryan** [comment]
The burn address for zksync lite is: 0x0000000000000000000000000000000000000000

**Comment by hyal1ngrumblu** [comment]
The user is requesting the burn address for useless coins on zksync lite on the Mainnet. The user has not encountered any specific issues or errors, and has not made any recent changes to the contract. They have tried searching for information online but have not found any answers.
